src/Microsoft.Data/Microsoft.DataFrames.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
Since this is only 1 TFM, you can use the `TargetFramework` property, not `TargetFrameworks`. You only need the plural when you have more than 1.